pkgs/development/haskell-modules/lib/default.nix

```
@@ -354,4 +354,6 @@ rec {
   # same package in the (recursive) dependencies of the package being
   # built. Will delay failures, if any, to compile time.
   allowInconsistentDependencies = compose.allowInconsistentDependencies;
+
+  incremental = pkgs: args: compose.incremental args pkgs;
```

Very small nitpick, but `pkgs` is currently only a single package, right?

```suggestion
  incremental = pkg: args: compose.incremental args pkg;
```